openmetadata-ui/src/main/resources/ui/src/pages/Configuration/EditUrlConfiguration/EditUrlConfigurationPage.tsx

```diff
@@ -182,14 +184,16 @@ const EditUrlConfigurationPage = () => {
 
   return (
     <ResizablePanels
-      className="content-height-with-resizable-panel"
+      className="content-height-with-resizable-panel m--t-sm"
       firstPanel={{
         children: firstPanelChildren,
         minWidth: 700,
         flex: 0.7,
         className: 'content-resizable-panel-container',
       }}
-      pageTitle={t('label.edit-entity', { entity: t('label.service') })}
+      pageTitle={t('label.edit-entity', {
+        entity: t('label.service'),
+      })}
```

Comment:
Same here, Let's scan through these, looks like we have wrong page titles in many pages